examples/prt4cmd/src/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.3)
```

Comment:
I noticed that CMake 3.3 does not yet include FetchContent, so our minimum required version is wrong. From the docs, it looks like FetchContent was added in [CMake 3.11](https://cmake.org/cmake/help/v3.11/release/3.11.html?highlight=changes#modules).

I suggest we raise the minimum to version [3.20](https://cmake.org/cmake/help/v3.20/release/3.20.html), it is not too old and this is the version included in the latest update of Visual Studio 2019:
```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --version

cmake version 3.20.21032501-MSVC_2
```